public/lib/dragAndDroplayercontrol/__tests__/DndLayerControl.js

```diff
@@ -175,6 +226,103 @@ describe('Kibi Enhanced Tilemap', () => {
 
         expect(_allLayers[2].id).to.eql(geoFiltersLayer.id);
       });
+    });
+    describe('_getLayerLevelConfig', () => {
+      it(`should assign default values`, () => {
```

Comment:
```suggestion
      it('should assign default values', () => {
```
please correct in all titles where string template is NOT needed